package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "riff-mm-webapp",
-  "version": "1.0.0-dev.6",
+  "version": "1.0.0-dev.7",
```

Comment:
I know you didn't make this change deliberately. You need to be careful about what files you commit.

Reply:
Sorry I've got too used to using `git add .`. I will be more careful in the future.